### ISSUE TITLE
\url{} is black

### ISSUE THREAD

**Issue by xworld21** [vlmantova/bookml#17]
`\url{}` generates black links (this is most likely caused by a LaTeXML v0.8.8 change in font handling which adds black colors to virtually every piece of text).

**Comment by hayden-MB** [comment]
Hello, just adding to this to say that in seipa mode the links are the same colour as other text.
